it/src/test/java/apoc/it/core/RBACOnLoadTest.java

```diff
@@ -137,6 +142,71 @@ public void testRBACOnDeny() throws IOException {
         }
     }
 
+    @Test
+    public void testRBACOnFileDeny() throws IOException {
```

Comment:
This isn't going to follow the file path though is it? (because it is http scheme)

Reply:
You're right, I'll remove it and add some other ones. Should have waited with tagging you until all the old tests were green at least...